### ISSUE TITLE
ipydeps.pip breaks when you pip install -e a package

### ISSUE THREAD

**Issue by kafonek** [nbgallery/ipydeps#32]
`pip install -e <package>` is a way to install libraries that you intend to do development on.  It creates a `<package>.egg-link` in site-packages and (I think?) adds the development directory to your PYTHONPATH.  When you run `pip freeze --all` after installing an 'editable' package, that package shows up as something like this:

```
arrow==0.12.1
-e git+git@gitserver.com:kafonek/package_name@githash#egg-package
asn1crypto==0.23.0
```

In `ipydeps._get_freeze_package_name`, it tries to split each package info on `==` and errors out when it hits a package installed by `pip install -e`.

**Comment by billallen256** [comment]
Interesting.  If only pip could be used as an API...

Filtering out the lines without a `==` should be fairly trivial.  I'll try to release a fix soon.

**Comment by billallen256** [comment]
Fixed in 0.8.1.
